opt_probs.py

```diff
@@ -289,7 +290,7 @@ def eval_node_probs(self):
                 subset = self.keep_sample[np.where(
                     self.keep_sample[:, parent[i - 1]] == j)[0]]
 
-                if len(subset) == 0:
+                if not subset:
```

Comment:
As above, this line is also causing an error when run on my machine. Could we also revert back here?
```suggestion
                if len(subset) == 0:
```